ov/id_pools_ipv4_range.go

```diff
@@ -193,116 +186,125 @@ func (c *OVClient) GetFreeFragments(filter string, sort string, start string, co
 	return freeFragments, nil
 }
 
-func (c *OVClient) CreateIPv4Range(ipv4 CreateIpv4Range) error {
+func (c *OVClient) CreateIPv4Range(ipv4 CreateIpv4Range) (Ipv4Range, error) {
 	log.Infof("Initializing creation of ipv4Range for %s.", ipv4.Name)
 	var (
-		uri = "/rest/id-pools/ipv4/ranges/"
-		t   = (&Task{}).NewProfileTask(c)
```

Comment:
since you have removed task, have you checked if no task uri is returned?

Reply:
This resource doesn't have any task.